<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OTP - Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="encryptTab">
       <attribute name="title">
        <string>Encrypt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="encryptFileLayout">
          <item>
           <layout class="QHBoxLayout" name="encryptButtonLayout">
            <item>
             <widget class="QLabel" name="encryptKeyfilesLabel">
              <property name="text">
               <string>Available keyfiles:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="encryptButtonSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="enDeleteButton">
              <property name="toolTip">
               <string>Delete Keyfile</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::EditDelete"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="enGenerateButton">
              <property name="toolTip">
               <string>Generate Keyfile</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="enExportButton">
              <property name="toolTip">
               <string>Export keyfile</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::GoUp"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="enImportButton">
              <property name="toolTip">
               <string>Import keyfile</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::GoDown"/>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="encryptKeyfilesView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="encryptLayout">
          <item>
           <widget class="QLabel" name="inputMessageLabel">
            <property name="text">
             <string>Input message:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="inputMessageField"/>
          </item>
          <item>
           <widget class="QPushButton" name="encryptButton">
            <property name="text">
             <string>Encrypt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="outputCiphertextField"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="decryptTab">
       <attribute name="title">
        <string>Decrypt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="decryptFileLayout">
          <item>
           <layout class="QHBoxLayout" name="decryptButtonLayout">
            <item>
             <widget class="QLabel" name="decryptKeyfilesLabel">
              <property name="text">
               <string>Available keyfiles:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="decryptButtonSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="deDeleteButton">
              <property name="toolTip">
               <string>Remove Keyfile</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListRemove"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="deAddButton">
              <property name="toolTip">
               <string>Add Keyfile</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListAdd"/>
              </property>
              <property name="arrowType">
               <enum>Qt::ArrowType::NoArrow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="decryptKeyfilesView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="decryptLayout">
          <item>
           <widget class="QLabel" name="inputCiphertextLabel">
            <property name="text">
             <string>Input ciphertext:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="inputCiphertextField"/>
          </item>
          <item>
           <widget class="QPushButton" name="decryptButton">
            <property name="text">
             <string>Decrypt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="outputMessageField"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
